### PR TITLE
DIABLO-122, /rooms view with select-capability per room; /room view; and /printable room view

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-122

Designs: https://jira.ets.berkeley.edu/jira/browse/DIABLO-98

Remove Salesforce-related code, too.

### CODE REVIEW

File: diablo/api/approval_controller.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def course_capture_sign_up():
     if Approval.get_approval(approved_by_uid, section_id, term_id):
         raise ForbiddenRequestError(f'You have already approved recording of {section["courseName"]}, {term_name}')
 
-    location = section['room']['location']
+    location = section['meetingLocation']
```
Comment (johncrossman) [inline diablo/api/approval_controller.py]: Room data comes from Diablo's own db so we merge it into the feed separately.

File: diablo/lib/berkeley.py [hunk 2]
```diff
@@ -23,30 +23,6 @@
...
-                    })
-            else:
-                key = 'presentation_audio'
-                capture_options.append({
-                    'text': RECORDING_TYPE_NAMES_PER_ID[key],
-                    'value': key,
-                })
-            break
-    return capture_options
-
```
Comment (johncrossman) [inline diablo/lib/berkeley.py]: We no longer pull room capabilities from Salesforce. The complex logic above is not necessary.

**Comment by paulkerschen** [comment]
The Force was inside us all along.

**Comment by johncrossman** [comment]
@paulkerschen - Thanks for the review. Ready to merge?

**Pull request merged**